src/app/data-planes/components/DataPlaneDetails.spec.ts

```diff
@@ -33,7 +33,7 @@ describe('DataPlaneDetails', () => {
 
   test('has expected content and UI elements', async () => {
     const expectedOverviewStrings = ['DPP: backend', 'test-mesh', 'online', 'kuma.io/service', 'backend', 'kuma.io/protocol', 'http', 'envoy', '1.16.2', 'kumaDp', '1.0.7', 'coredns', '1.8.3']
-    const expectedInsightStrings = ['Connect time: February 17, 2021 at 7:33:36 AM', 'Control Plane Instance ID', 'Disconnect time: February 17, 2021 at 7:33:36 AM']
+    const expectedInsightStrings = ['Connect time: February 17, 2021 at 7:33:36', 'CP instance ID', 'Disconnect time: February 17, 2021 at 7:33:36']
```

Comment:
Looks like we lost the `AM` here?

Reply:
On purpose. This is one of the tests that broke with the two breaking changes in Node.js 18.13 and 18.15 where the space before `AM` or `PM` was changed. I’ve removed it here because asserting for the presence of `AM` as part of the string doesn’t improve the quality of the tests and removing it _does_ avoid it from breaking again once we upgrade past Node.js 18.15.